pages.pt-BR/common/7za.md

```diff
@@ -0,0 +1,25 @@
+# 7za
+
+> Um compactador de arquivos com alta taxa de compressão.
+> Versão do `7z` com suporte para poucos tipos de arquivamento/compressão.
+> Página oficial: <https://www.7-zip.org/>.
+
+- Compactar um arquivo ou diretório:
+
+`7za a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretorio}}`
```

Comment:
I don't think we should avoid accented characters, especially since paths and filenames can contain them, anyway. I'd therefore have used "diretório", for correctness. Thoughts?

Reply:
Sure.